README.md

```diff
@@ -98,8 +98,12 @@ The docs themselves are published to GitHub Pages and live at [http://docs.vanil
 	* OS X: `$ brew install yarn`
 	* Windows: ???
 2. Fork or clone the repository (depending on whether you have commit access)
-3. Install Hugo `brew install hugo` [https://gohugo.io/getting-started/installing/](https://gohugo.io/getting-started/installing/) OR if you already have hugo, run `brew upgrade hugo`
-4. From the root of the folder, use `yarn` to install the project: `$ yarn install`
+3. Install Hugo `v0.29`. The site does not currently function properly with newer versions of hugo.
```

Comment:
Add details for mac users, specifically downloading "hugo_0.29_macOS-64bit.tar.gz" and copying hugo to `/usr/local/bin`. Technically, this repo is open source, but since we're all on mac internally, I would add those details.